.lighthouse/jenkins-x/pullrequest.yaml

```diff
@@ -27,6 +27,30 @@ spec:
           resources: {}
         - name: build-make-test
           resources: {}
+        # Move to catalog
+        - image: golang:1.17.9-alpine3.15
```

Comment:
Before moving it to catalog, want to test and see if this works first.

Reply:
This seems quite cool. Although are we looking for a way to configure a target for new code?

Be nice to see the coverage on new code -- seems it's not working currently?